sass/layout/hero.sass

```diff
@@ -128,6 +128,13 @@
           .nav-item
             border-top-color: rgba($color-invert, 0.2)
   // Sizes
+  &.is-small
+    +tablet
+      .hero-body
+        padding-bottom: 1rem
+        padding-top: 1rem
+      .container 
+        margin: 0 auto;
```

Comment:
I've done this in my projects, as when the viewport get's to tablet size, the container loses its center alignment. This keeps it center at the smaller breakpoint. I dare say that is why its been implemented in this PR